rust/src/write.rs

```diff
@@ -126,20 +128,34 @@ impl WriteOptions {
         Self::default()
     }
 
+    /// Specifies the compression that should be used on chunks.
     pub fn compression(self, compression: Option<Compression>) -> Self {
         Self {
             compression,
             ..self
         }
     }
 
+    /// specifies the profile that should be written to the MCAP Header record.
     pub fn profile<S: Into<String>>(self, profile: S) -> Self {
         Self {
             profile: profile.into(),
             ..self
         }
     }
 
+    /// specifies the target size of each chunk. Messages will be written to chunks
+    /// until the chunk size is larger than the target chunk size, at which point the
```

Comment:
Nit:

```suggestion
    /// specifies the target size of each chunk.
    ///
    /// Messages will be written to chunks until the chunk size
    /// is larger than the target chunk size, at which point the
```

Otherwise rustdoc will put the whole paragraph in the "brief" description.